lib/iris/coords.py

```diff
@@ -1940,7 +1940,13 @@ def _new_points_requirements(self, points):
         if points.size > 1 and not iris.util.monotonic(points, strict=True):
             raise ValueError('The points array must be strictly monotonic.')
 
-    def _points_setter(self, points):
+    @property
+    def points(self):
+        # Call the parent points getter.
```

Comment:
I'm now finding that the `points` and `bounds` properties of DimCoords now don't have a useable docstring, whereas previously they did, because we were only replacing the *setter*, as in `xxx = property(...Coord._xxx_getter, _xxx_setter)`.

By my experiment, the same occurs with `help()` and `print(coord.xxx.__doc__)` : 

Results before:
```
>>> print(iris.coords.Coord.points.__doc__) 
The coordinate points values as a NumPy array.
>>> print(iris.coords.Coord.bounds.__doc__)

        The coordinate bounds values, as a NumPy array,
        or None if no bound values are defined.

        .. note:: The shape of the bound array should be: ``points.shape +
            (n_bounds, )``.

        
>>> print(iris.coords.DimCoord.points.__doc__)
The coordinate points values as a NumPy array.
>>> 
>>> iris.coords.DimCoord.bounds.__doc__ is iris.coords.Coord.bounds.__doc__
True
>>> iris.coords.DimCoord.points.__doc__ is iris.coords.Coord.points.__doc__
True
>>> 
```

Results new:
```
>>> print(iris.coords.Coord.points.__doc__)
The coordinate points values as a NumPy array.
>>> print(iris.coords.Coord.bounds.__doc__)

        The coordinate bounds values, as a NumPy array,
        or None if no bound values are defined.

        .. note:: The shape of the bound array should be: ``points.shape +
            (n_bounds, )``.

        
>>> print(iris.coords.DimCoord.points.__doc__)
None
>>> print(iris.coords.DimCoord.bounds.__doc__)
None
>>> 
```

I don't know if you can achieve this with a wrap, or what ...

Reply:
Actually, I also goofed ...
```
>>> print(iris.fileformats.pp.PPField2.t1.__doc__)
A cftime.datetime object consisting of the lbyr, lbmon,
>>> 
```

As written, there are 2 strings.  It obviously doesn't merge them, but only takes the first one as the docstring.  PPField3, by constrast, is okay.  I will fix ...